hail/src/main/scala/is/hail/io/fs/RouterFS.scala

```diff
@@ -15,6 +23,47 @@ case class RouterFSURL private (_url: FSURL, val fs: FS) extends FSURL {
   override def toString(): String = url.toString
 }
 
+case class CloudStorageFSConfig(
```

Comment:
the plan is to send this over from python to the scala backend.
doing so will remove need to pass requester pays info via feature flags.